packages/angular_devkit/build_angular/src/angular-cli-files/plugins/karma.ts

```diff
@@ -240,6 +240,9 @@ const eventReporter: any = function (this: any, baseReporterDecorator: any) {
       failureCb && failureCb();
     }
   }
+
+  // avoid duplicate failure message
+  this.specFailure = (_browser: any, result: any) => {};
```

Comment:
This can be just `this.specFailure = () => {};`, and the same applies to the other ones.

Reply:
done, thanks